### PR TITLE
Add description comments into schematypes.go

### PR DESCRIPTION
**Pull request by eliben**
Updates #4

### CODE REVIEW

File: cmd/gentypes/gentypes.go [hunk 1]
```diff
@@ -173,6 +173,19 @@ func emitToplevelType(typeName string, descJson json.RawMessage) string {
 		log.Fatal(err)
 	}
 
+	var comment string
+	descDescriptionJson, ok := descMap["description"]
```
Comment (polinasok) [inline cmd/gentypes/gentypes.go]: descriptionJson would be more consistent with line 166 and easier to read.

File: cmd/gentypes/gentypes.go [hunk 2]
```diff
@@ -173,6 +173,19 @@ func emitToplevelType(typeName string, descJson json.RawMessage) string {
...
+	if ok {
+		if err := json.Unmarshal(descDescriptionJson, &comment); err != nil {
+			log.Fatal(err)
+		}
+	}
+
+	if len(comment) > 0 {
+		comment = prependCommentPerLine(fmt.Sprintf("%s: %s", typeName, comment))
```
Comment (polinasok) [inline cmd/gentypes/gentypes.go]: Based on the name, I thought this prepended the comment before the type definition, so I was pretty confused at first. Suggest renaming: MakeIntoGoComment, CommentOutEachLine. If you really want to emphasize that you are prepending forward slashes to each line, then maybe call prependSlashesPerLine?

File: cmd/gentypes/gentypes.go (hunk 1 above)
Comment (eliben) [reply cmd/gentypes/gentypes.go]: Done

File: cmd/gentypes/gentypes.go (hunk 2 above)
Comment (eliben) [reply cmd/gentypes/gentypes.go]: Done

**Pull request merged**